### PR TITLE
Add method to retrieve page access tokens.

### PR DESCRIPTION
**Pull request by Smamaxs**
Tested/

It needed in case when I want to publish something on my Page's wall

Why. Look here:
http://stackoverflow.com/a/23675226/3048464

### CODE REVIEW

File: facebook/__init__.py [hunk 1]
```diff
@@ -130,6 +130,18 @@ def put_object(self, parent_object, connection_name, **data):
         return self.request(parent_object + "/" + connection_name,
                             post_args=data,
                             method="POST")
+                            
+    def get_access_token_for_page(self, profile_id="me", page_id=None):
+        """
+        Retrive access token of user as admin of Page
```
Comment (jaddison) [inline facebook/__init__.py]: Minor spelling: `Retrieve` instead of `Retrive`

**Comment by Smamaxs** [comment]
I maked needed changes

**Comment by martey** [comment]
@Smamaxs, I re-reviewed your most recent changes and noticed that they still do not include updated documentation and automated test cases to cover the new method. Can you reply and let everyone whether you are still working on this?

File: facebook/__init__.py [hunk 2]
```diff
@@ -163,10 +163,44 @@ def put_object(self, parent_object, connection_name, **data):
...
+        """
+        if not page_id:
+            return None
+        response = self.request(profile_id + "/accounts")
+        for token in response['data']:
+            if str(page_id) in token['id']:
+                return token['access_token']
+        return None
+
+    def put_wall_post(self, message, attachment={}, profile_id="me"):
```
Comment (martey) [inline facebook/__init__.py]: The `put_wall_post` method was removed in 88dbc2f33ec566872188fbad38d8677e4489f5ec; I think the last merge comment re-added it by mistake.

**Pull request closed without merging**